src/str_extract.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os, sys, re
+import argparse
+
+def strextract(dir, suffix=None, path=False, all=False):  
+    # Expression régulière pour trouver des chaînes littérales entre guillemets simples ou doubles
```

Comment:
In english, please

Reply:
If I translate this very comment I should translate all the other ones, isnt't it?

Reply:
Of course :)